### DISCUSSION TITLE
在WSL2中不支持中文路径与中文名(划掉) 什么问题也没有

### DISCUSSION THREAD

**Discussion by Mythos-404** [xmake-io/xmake#2848] [Q&A]
哎又是我

如题

环境
![image](https://user-images.githubusercontent.com/96325829/191405151-eaab2b1f-56d3-433a-a567-46a8801654f4.png)

xmake 版本

![image](https://user-images.githubusercontent.com/96325829/191405438-5478ef16-a2b9-4542-9d67-59412b61a0b4.png)

xmake.lua 代码
```lua
add_rules("mode.debug")
set_languages("c++20")
set_toolchains("llvm")

target("cxx")
    set_kind("binary")
    add_ldflags("-stdlib=libc++")
    add_cxxflags("-stdlib=libc++", "-Wfatal-errors", "-Wall", "-Wextra", "-Wshadow")
    add_includedirs("src")
    set_pcxxheader("src/stdafx.h")
    -- add_files("src/file/01.cpp")
    set_filename("src/9内存模型和名称空间/practise/golf.cpp", "src/9内存模型和名称空间/practise/paly.cpp")
    -- add_files("src/**.cpp")
```

之前没问题的

对了在windows没有问题

又得麻烦作者您了了

`set_filename("src/9内存模型和名称空间/practise/golf.cpp", "src/9内存模型和名称空间/practise/paly.cpp")` 我这句写错了

我是傻逼

**Comment by waruqi** [comment]
? 不知道你要搞啥，有问题 说清楚，并提供 logs

**Reply by Mythos-404** [reply]
能给我删了吗

我犯傻了

把添加文件`add_files()` 写成 ` set_filename()`
